src/fragments/lib-v1/graphqlapi/flutter/advanced-workflows/30_nested.mdx

```diff
@@ -1,6 +1,6 @@
 ```dart
 const getPost = 'getPost';
-String graphQLDocument = '''query GetPost(\$id: ID!) {
+const graphQLDocument = '''query GetPost(\$id: ID!) {
```

Comment:
shouldn't this be "final" instead of "const"?

Reply:
`const` is correct here. A linter rule we observe [prefer_const_declarations](https://dart.dev/tools/linter-rules/prefer_const_declarations), which prefers `const` when initializing a variable with a static value reinforced this.